frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -175,14 +175,15 @@ class PopupCreateFile extends React.Component {
 
     this.setState({ uploadStarted: true })
 
-    Promise.all(state.fileToUploadList.map((file) => this.postFile(file)))
-      .then(xhrList => {
-        this.handleAllFileUploadEnd(xhrList.map((xhr, index) => this.handleFileUploadEnd(xhr, state.fileToUploadList[index])))
-      })
-      .catch(() => {
-        this.sendGlobalFlashMessage(props.t('Error while creating file'))
-        this.handleClose(false)
-      })
+    Promise.all(state.fileToUploadList.map(async (file) => ({
+      xhr: await this.postFile(file),
```

Comment:
If you put an "await" here the next file will only be sent once the current file has finished.
So the promise.all doesn't do anything.
I'm not entirely sure about this since I'm not doing a full review.
Manually test it properly to check that multiples files a sent in parallel

Reply:
@metzgegu you must fix this next week in order we can deliver 2.6.0 next friday

Reply:
It works properly because the `await` is inside a object initialization, so it will initialize objects with map and wait until objects are created entirely with `Promise.all`, 
but I will use an `async` function to build this object in order to clarify the code